app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -150,18 +161,20 @@ params:
       default: sliding
       value_in_examples:
       description: |
-        This sets the time window to either `sliding` or `fixed`
+        This sets the time window to either `sliding` or `fixed`.
   extra: |
     > Note:  Redis configuration values are ignored if the `cluster` strategy is used.
 
-    **Note: PostgreSQL 9.5+ is required when using the `cluster` strategy with `postgres` as the backing Kong cluster data store. This requirement varies from the PostgreSQL 9.4+ requirement as described in the <a href="/install/source">Kong Community Edition documentation</a>.**
+    **Notes:**
 
-    **Note: The `dictionary_name` directive was added to prevent the usage of the `kong` shared dictionary, which could lead to `no memory` errors**
+     * PostgreSQL 9.5+ is required when using the `cluster` strategy with `postgres` as the backing Kong cluster data store. This requirement varies from the PostgreSQL 9.4+ requirement as described in the <a href="/install/source">Kong Community Edition documentation</a>.
+
+     * The `dictionary_name` directive was added to prevent the usage of the `kong` shared dictionary, which could lead to `no memory` errors.
 
 ---
 
 **GraphQL Rate Limiting Advanced** is an extension of
```

Comment:
This description is a bit cleaner/more grammatically correct than the description at the top of the topic. Maybe use this up above? I also don't know why we'd need the description again down here.

Reply:
> This description is a bit cleaner/more grammatically correct than the description at the top of the topic. Maybe use this up above? I also don't know why we'd need the description again down here.

All I did was split out the note and remove the bold. I'd rather leave it be for now in the interest of time.